.demo/env/charts/cert-manager.yaml

```diff
@@ -1,5 +1,6 @@
 charts:
     cert-manager:
+        _rawValues: {}
```

Comment:
I don't think we should add this line to the demo values as this is the default. Please add at least one non-empty `_rawValues` entry to one of the charts, both for coverage, but also as an example.
This is an expert setting and we don't want to put it in every file for others to start thinking, hey, lets just use that.